equinix/resource_network_device.go

```diff
@@ -1368,6 +1382,7 @@ func updateNetworkDeviceResource(primary *ne.Device, secondary *ne.Device, d *sc
 func flattenNetworkDeviceSecondary(device *ne.Device) interface{} {
 	transformed := make(map[string]interface{})
 	transformed[neDeviceSchemaNames["UUID"]] = device.UUID
+	transformed[neDeviceSchemaNames["Tier"]] = device.Tier
```

Comment:
In HA case, does tier apply to both primary and secondary? Can we use different tiers for primary and secondary device?

Reply:
no, we cant use different for primary and secondary